constants/links.ts

```diff
@@ -1,6 +1,6 @@
 import { CurrencyKey } from './currency';
 
-export const PROD_HOSTNAME = 'kwenta.io';
+export const PROD_HOSTNAME = 'kwenta.eth.limo';
```

Comment:
Not 100% sure this is fine/needed or doesn't break any dev or CI flows

Reply:
this won't cause any problem, it's only used in one place in case the system is under maintenance